packages/app/cypress/e2e/subscriptions/specChange-subscription.cy.ts

```diff
@@ -0,0 +1,357 @@
+
+        cy.get('[data-testid="spec-file-item"]')
+        .should('have.length', 4)
+        .should('contain', 'blank-contents.spec.js')
+        .should('contain', 'dom-container.spec.js')
+        .should('contain', 'dom-content.spec.js')
+        .should('contain', 'dom-list.spec.js')
+
+        cy.withCtx(async (ctx, o) => {
+          await ctx.actions.file.removeFileInProject(o.path)
+        }, { path: getPathForPlatform('cypress/e2e/dom-list.spec.js') })
+
+        cy.get('[data-testid="spec-file-item"]')
+        .should('have.length', 3)
+        .should('contain', 'blank-contents.spec.js')
+        .should('contain', 'dom-container.spec.js')
+        .should('contain', 'dom-content.spec.js')
+      })
+
+      it('handles removing the last file', () => {
+        cy.contains('dom-content.spec').click()
+        cy.get('[data-model-state="passed"]').should('contain', 'renders the test content')
+
+        cy.withCtx(async (ctx, o) => {
+          await Promise.all(o.paths.map((path) => ctx.actions.file.removeFileInProject(path)))
+        }, {
+          paths: [
+            getPathForPlatform('cypress/e2e/blank-contents.spec.js'),
+            getPathForPlatform('cypress/e2e/dom-container.spec.js'),
+            getPathForPlatform('cypress/e2e/dom-list.spec.js'),
+          ],
+        })
+
+        cy.get('[data-testid="spec-file-item"]')
+        .should('have.length', 1)
+        .should('contain', 'dom-content.spec.js')
+
+        cy.withCtx(async (ctx, o) => {
+          await ctx.actions.file.removeFileInProject(o.path)
+        }, { path: getPathForPlatform('cypress/e2e/dom-content.spec.js') })
+
+        cy.get('[data-cy="create-spec-page-title"]')
+        .should('contain', defaultMessages.createSpec.page.customPatternNoSpecs.title)
+      })
+
+      it('responds to a cypress.config.js file change', () => {
+        cy.contains('dom-content.spec').click()
+        cy.get('[data-model-state="passed"]').should('contain', 'renders the test content')
+
+        cy.get('[data-testid="spec-file-item"]')
+        .should('have.length', 4)
+        .should('contain', 'blank-contents.spec.js')
+        .should('contain', 'dom-container.spec.js')
+        .should('contain', 'dom-content.spec.js')
+        .should('contain', 'dom-list.spec.js')
+
+        cy.withCtx(async (ctx, o) => {
+          await ctx.actions.file.writeFileInProject('cypress.config.js',
+`const { devServer } = require('@cypress/react/plugins/load-webpack')
+    
+module.exports = {
+  projectId: 'abc123',
+  experimentalInteractiveRunEvents: true,
+  component: {
+    specPattern: 'src/**/*.{spec,cy}.{js,ts,tsx,jsx}',
+    supportFile: false,
+    devServer,
+    devServerConfig: {
+      webpackFilename: 'webpack.config.js',
+    },
+  },
+  e2e: {
+    specPattern: 'cypress/e2e/**/dom-cont*.spec.{js,ts}',
+    supportFile: false,
+  },
+}`)
+        }, { path: getPathForPlatform('cypress/e2e/dom-list.spec.js') })
```

Comment:
Doesn't look like the path is being used here.

Reply:
[`574a705` (#20657)](https://github.com/cypress-io/cypress/pull/20657/commits/574a705437ef800269ff00682da405dfe5c858c2)